### PR TITLE
added homepage and Video UI

### PR DESCRIPTION
**Pull request by cjinc121**
- added Header,sidebar,category row UI
- made them responsive 
Screenshot (157)
Screenshot (158)
-added videos 
-added videoplayer and comment ui
-made them responsive for mobile and desktop
Screenshot (162)

Screenshot (163)
Screenshot (161)

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Overall the code looks fine

File: BingeIT/src/components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+.user{
+  color:var(--text-color);
+  align-items: center;
+ display: flex;
+ gap: 1rem;
+font-size: 1.5rem;
+
+}
+.user-icon{
+cursor: pointer;
```
Comment (paarthbajaj) [inline BingeIT/src/components/Header/Header.css:57]: Could have used utility class for cursor pointer, which could save some lines of repeating code.

File: BingeIT/src/components/Sidebar/Sidebar.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.sidebar{
+  background-color:var(--bg-color2);
+  position: relative;
+  width:15%;
+  height: 100%;
+  position:fixed;
+}
+.sidebar-item{
+  display: flex;
```
Comment (paarthbajaj) [inline BingeIT/src/components/Sidebar/Sidebar.css:9]: Could have used a utility class for display flex, which could save some lines of repeating code.

File: BingeIT/src/index.css [hunk 3]
```diff
@@ -1,13 +1,39 @@
...
+  }
+
+  ::-webkit-scrollbar-thumb {
+    background-color: var(--color-hover);
+    border-radius: 20px;
+    border: 6px solid transparent;
+    background-clip: content-box;
+  }
+
+  ::-webkit-scrollbar-thumb:hover {
```
Comment (paarthbajaj) [inline BingeIT/src/index.css:38]: I liked how you customized the scrollbar of the app. Really good efforts.

**Review by paarthbajaj — APPROVED** [review]
Gave some suggestions, otherwise, the code is very well written. Great Efforts. You can merge.

**Review by SukanyaSen530 — APPROVED** [review]
Look's good

**Pull request merged**